SonarQube.Bootstrapper/Program.cs

```diff
@@ -100,6 +101,13 @@ private static int PreProcess(IBuildAgentUpdater updater, IBootstrapperSettings
 
         private static int PostProcess(IBootstrapperSettings settings, ILogger logger)
         {
+
+            if (!File.Exists(settings.PostProcessorFilePath))
```

Comment:
Needs a test. There's plenty of infrastructure in BootstrapperExeTests - you can check that the execution fails that that errors are logged.